yarn.lock

```diff
@@ -4329,6 +4329,11 @@ ramda@^0.26.1:
   resolved "https://registry.yarnpkg.com/ramda/-/ramda-0.26.1.tgz#8d41351eb8111c55353617fc3bbffad8e4d35d06"
   integrity sha512-hLWjpy7EnsDBb0p+Z3B7rPi3GDeRG5ZtiI33kJhTt+ORCd38AbAIjB/9zRIUoeTbE/AVX5ZkU7m6bznsvrf8eQ==
 
+ramda@^0.27.0:
```

Comment:
you have 2 versions of ramda in this yarn.lock

Reply:
looks like 3 😱 .

`0.26.1` is coming from my `@rjhilgefort/export-dir`. Would guess you'd have to upgrade there if you really wanted to get to 1 version in here. I don't think I have the time this weekend to do the upgrade and test things. I'd accept anything you guys want to change with it though if you toss me a PR over there.

Didn't track down where `0.18.0` is coming from. You can use `yarn why` to figure it out:
https://classic.yarnpkg.com/en/docs/cli/why/

Reply:
https://github.com/rjhilgefort/export-dir/pull/11 

Rob, I opened this for you on friday. 😛

Reply:
0.18.0 is from treis which is a dev dependency so that one we can leave alone. 

```
=> Found "ramda@0.27.0"
info Has been hoisted to "ramda"
info This module exists because it's specified in "dependencies".
info Disk size without dependencies: "3.05MB"
info Disk size with unique dependencies: "3.05MB"
info Disk size with transitive dependencies: "3.05MB"
info Number of shared dependencies: 0
=> Found "treis#ramda@0.18.0"
info This module exists because "treis" depends on it.
info Disk size without dependencies: "1.48MB"
info Disk size with unique dependencies: "1.48MB"
info Disk size with transitive dependencies: "1.48MB"
info Number of shared dependencies: 0
=> Found "@rjhilgefort/export-dir#ramda@0.26.1"
info This module exists because "@rjhilgefort#export-dir" depends on it.
info Disk size without dependencies: "3.01MB"
info Disk size with unique dependencies: "3.01MB"
info Disk size with transitive dependencie
```

PS. thanks for the tip on yarn why!

Reply:
> 0.18.0 is from treis which is a dev dependency so that one we can leave alone.

Sounds good!

> rjhilgefort/export-dir#11
>
> Rob, I opened this for you on friday. 😛

Merged! `1.1.13` is published
https://www.npmjs.com/package/@rjhilgefort/export-dir